### DISCUSSION TITLE
Appearance of Codeblock - colored left border

### DISCUSSION THREAD

**Discussion by M4rtyx** [mayurankv/Obsidian-Code-Styler#254] [Q&A] [answered]
Hello,

I have question regarded section:

> ### Appearance
> **Codeblock**
> Codeblocks can have their curvature changed in settings to make them appear more or less rounded.
> 
> They can also have coloured left borders based on the language (colours match the language icon) if enabled in settings. The width of this border can also be changed.
> 
> ![image](https://github.com/mayurankv/Obsidian-Code-Styler/assets/167995003/c0b16de2-7dcb-4fff-aa33-1fe4d672d12a)

Where can I set this colored left border based on the language? I was checking almost everything in settings but I didn't find an option where can I enable it. Can you please help me with that?

Thank you very much and have a nice day!

**Comment by d-s-dc** [comment, marked as answer]
Follow these steps
1. In the settings, select `Codeblock Styling`.
<img width="543" alt="image" src="https://github.com/mayurankv/Obsidian-Code-Styler/assets/68206552/a2430059-93f7-4005-a0c5-7c8cbe2c5614">

2. Then turn on `Language Coloured Borders`.
<img width="559" alt="image" src="https://github.com/mayurankv/Obsidian-Code-Styler/assets/68206552/0221bc78-479c-44d2-bb7a-c7c014a7a9dc">

**Reply by M4rtyx** [reply]
I am blind :D idk why I didn't see that before. But thank you very much for answer and help!
